website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -17,6 +17,7 @@ User accounts belong to individual people. Each user can be part of one or more
 ## API
 Use the [Account API](/cloud-docs/api-docs/account) to get account details, update account information, and change your password.
 
+<!-- BEGIN: TFC:only -->
 ## Log In With Your HashiCorp Cloud Platform Account
 
 We recommend using a [HashiCorp Cloud Platform (HCP)](https://portal.cloud.hashicorp.com/sign-up) account to log in to Terraform Cloud. Your HCP Account grants access to every HashiCorp product, HashiCorp Learn, and the Terraform Registry. If you use your HCP Account, you will manage user settings like multi-factor authentication and password resets from within HCP instead of the Terraform Cloud UI.
```

Comment:
```suggestion
We recommend using a [HashiCorp Cloud Platform (HCP)](https://portal.cloud.hashicorp.com/sign-up) account to log in to Terraform Cloud. Your HCP Account grants access to every HashiCorp product, HashiCorp Learn, and the Terraform Registry. If you use your HCP Account, you can manage user settings like multi-factor authentication and password resets from within HCP instead of the Terraform Cloud UI.
```

Suggest changing "will" to "can" to avoid future tense